LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ConcentratorDeduplication.cs

```diff
@@ -8,20 +8,29 @@ namespace LoRaWan.NetworkServer
     using System;
     using System.Buffers.Binary;
     using System.Security.Cryptography;
-    using System.Text;
-    using BasicsStation;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using LoRaTools.LoRaMessage;
     using Microsoft.Extensions.Caching.Memory;
     using Microsoft.Extensions.Logging;
 
-    internal sealed class ConcentratorDeduplication<T> :
-        IConcentratorDeduplication<T>, IDisposable
-        where T : class
+    public sealed class ConcentratorDeduplication :
+        IConcentratorDeduplication, IDisposable
     {
+        public enum Result
```

Comment:
we should not nest this. It is part of the interface (the result is returned), hence it should not be nested in a specific implementation.

Reply:
Done in 0671de319568b8c7273d9f0039c82f9df9c1660e